datarepo-clienttests/src/main/java/scripts/testscripts/RetrieveSnapshot.java

```diff
@@ -157,7 +157,7 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
   }
 
   public void userJourney(TestUserSpecification testUser) throws Exception {
```

Comment:
I think we're due a bigger refactor that doesn't require that you pass in the test user. Or that you can specify that you don't want a random test user.

Reply:
I agree...it makes these tests wicked hard to debug